apps/web/components/eventtype/RecurringEventController.tsx

```diff
@@ -1,8 +1,9 @@
-import type { EventTypeSetup, FormValues } from "pages/event-types/[type]";
+import type { EventTypeSetup } from "pages/event-types/[type]";
 import { useState } from "react";
 import { useFormContext } from "react-hook-form";
 
 import useLockedFieldsManager from "@calcom/features/ee/managed-event-types/hooks/useLockedFieldsManager";
+import type { FormValues } from "@calcom/features/eventtypes/lib/types";
```

Comment:
Moved FormValues to the features/lib to be able to use it from there too and avoid crossing boundaries between web app and the library